public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -298,17 +300,7 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
           ]
         : metricsByPrefix;
 
-    // Hint for big disabled lookups
```

Comment:
This fixes the hint as here it could end up null and rewriting any existing hints in state.